L1Trigger/L1THGCal/test/testHGCalL1T_RelVal_cfg.py

```diff
@@ -66,6 +66,18 @@
 process.load('L1Trigger.L1THGCal.hgcalTriggerPrimitives_cff')
 process.hgcl1tpg_step = cms.Path(process.hgcalTriggerPrimitives)
 
+from L1Trigger.L1THGCal.customTriggerCellSelect import custom_triggercellselect_supertriggercell, custom_triggercellselect_threshold
```

Comment:
The following should be removed from the default test file. For personal tests it is better to have a separate private file.